torch/fx/symbolic_trace.py

```diff
@@ -2,6 +2,7 @@
 from types import CodeType, FunctionType
 from typing import Any, Dict, Optional, List, Callable, Union
 import torch
+from torch._C import ScriptObject  # type: ignore
```

Comment:
What was the lint error here?

Reply:
it's Error: Module 'torch._C' has no attribute 'ScriptModule'  [attr-defined]

Tried to add it in the mypy.ini for ignore import error, but it didn't work. A little bit hesitate to mark it ignore all the errors. So just mark this linne.